src/types.rs

```diff
@@ -40,8 +40,10 @@ pub struct HyperParameters {
     /// Threshold below which nodes are pruned in the first
     /// phase of the algorithm.
     pruning_threshold: f64,
-    /// How often do random walks return to the starting nodes.
-    damping_factor: f64,
+    /// How often do random walks on project nodes return to the starting nodes.
+    project_damping_factor: f64,
+    /// How often do random walks on user nodes return to the starting nodes.
+    user_damping_factor: f64,
```

Comment:
Please also adjust the comments to reflect the definition of damping factors in the oscoin whitepaper and the osrank data model documentation. The damping factor is the probability that the walk continues, `1 - damping_factor` that it terminates ie returns to the starting nodes.

Reply:
Fixed!